10/umbraco-cms/reference/notifications/contentservice-notifications.md

```diff
@@ -39,6 +39,8 @@ namespace Umbraco.Docs.Samples.Web.Notifications
 }
 ```
 
+The handler will also need to be registered, see https://docs.umbraco.com/umbraco-cms/reference/notifications#registering-notification-handlers 
```

Comment:
```suggestion
The handler will also need to be registered. See the [Notifications article](https://docs.umbraco.com/umbraco-cms/reference/notifications#registering-notification-handlers) for specifics on how to do this.
```

Reply:
Ah yes, I thought there'd be a nicer way to format the link! Thank you